x-pack/plugins/ml/public/services/results_service.js

```diff
@@ -1491,7 +1493,7 @@ function getEventDistributionData(
     }
 
     if (metricFieldName !== undefined && metricFieldName !== '') {
-      body.aggs.byTime.aggs = {};
+      body.aggs.byTime.aggs.sample.aggs.entities.aggs = {};
```

Comment:
these are some pretty long chains.
i can't believe i'm suggesting this, but would using `_.get` make this safer?

Reply:
I think it's safe because that structure is created in the same file just above as a single object assignment, that structure isn't created dynamically, here's a link to the full file: https://github.com/elastic/kibana/blob/33cbdb311701a74d3efe35ce7139d6a24351d0cc/x-pack/plugins/ml/public/services/results_service.js#L1453

Reply:
ah ok cool. i didn't spot that above.